ocs_ci/ocs/ui/views.py

```diff
@@ -240,10 +244,23 @@
         By.CSS_SELECTOR,
     ),
     "confirm_pvc_deletion": ('button[data-test="confirm-action"]', By.CSS_SELECTOR),
-    "pvc_test": ('a[data-test-id="test-pvc-fs"]', By.CSS_SELECTOR),
     "search_pvc": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
 }
 
+pvc_4_7 = {
+    "test-pvc-fs": ('a[data-test-id="test-pvc-fs"]', By.CSS_SELECTOR),
```

Comment:
@am-agrawa Is there `data-test-id` attribute for `test-pvc-fs` element?

Reply:
It was added by you for OCP 4.7 and has changed in 4.8.